### ISSUE TITLE
Opening a file in a directory fails or is unintuitive

### ISSUE THREAD

**Issue by tomnewport** [radical-cybertools/saga-python#523]
I have a directory object, ```remote_dir```, which contains a log file:

```python
>>> for x in remote_dir.list():
>>>     print x
test_job.log
```

The documentation is unclear about how the ```open``` method of ```Directory``` works, however I tried this:

```python
>>> remote_dir.open("test_job.log")

DoesNotExist: File does not exist: '/test_job.log' -  (/saga/adaptors/shell/shell_file.py +1056 (initialize)  :  raise saga.DoesNotExist("File does not exist: '%s' - %s" % (self.url.path, out)))

```

What would the correct usage have been?

**Comment by andre-merzky** [comment]
Hi Tom, 

I think you hit the same problem as in #520.  Specifying an absolute path on open should work -- but the code you use is supposed to work, too, and thus needs fixing.

If its ok for you, please close this ticket in favor of #520.

Best, Andre.

**Comment by tomnewport** [comment]
Thanks Andre - closed in favor of #520.
